### PR TITLE
ScrollUp, search and sorting button

### PR DESCRIPTION
**Pull request by Sanae-G**

### CODE REVIEW

File: client/src/components/SortButton.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import ChronologicalOrderIcon from '../icons/chronological';
+import AntichronologicalOrderIcon from '../icons/antichronological';
+
+function SortButton({ chronological, setChronological }) {
+  function handleChronology() {
+    setChronological(!chronological);
+  }
+
+  console.log(chronological);
```
Comment (anitaschiller) [inline client/src/components/SortButton.js:10]: There's a leftover console.log!

File: client/src/components/SearchInput.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import SearchIcon from '../icons/search';
+import CancelationIcon from '../icons/cancelIcon';
+
+function SearchInput({ moments, setMoments, setFilteredPosts }) {
+  // const [searchButtonState, setSearchButtonState] = useState(false);
```
Comment (anitaschiller) [inline client/src/components/SearchInput.js:7]: Can be deleted if you don't use it anymore!

File: client/src/components/ScrollUpButton.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > 300) {
+        setButtonState(true);
+      } else {
+        setButtonState(false);
+      }
+    });
+  }, []);
```
Comment (anitaschiller) [inline client/src/components/ScrollUpButton.js:17]: I couldn't try it unfortunately (would be easier if the app was deployed 😉) but what happens if you scroll up and down without doing a reload in between? Does the button re-appear when you scroll below 300 on the y-axis?

File: client/src/components/ScrollUpButton.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+// import styled from 'styled-components';
```
Comment (anitaschiller) [inline client/src/components/ScrollUpButton.js:1]: Can be deleted!

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Hi Sanae! 
I left some comments but overall a good job 😊. Did you already think about deploying your app? Would be easier to review and try out stuff.

File: client/src/components/Header/JournalHeader.js [hunk 5]
```diff
@@ -1,13 +1,50 @@
...
-    return (
-        <Header color={color}>
-            <h1>{heading}</h1>
-            <p>{period}</p>
-        </Header>
-        );
+export default JournalHeader;
+
+export const Header = styled.header`
+  background-color: ${({ color }) => color};
```
[the remark this answers is not in the document]
Comment (Sanae-G) [reply client/src/components/Header/JournalHeader.js:21]: This is for storybook. There you can change the color of the header.

File: client/src/components/PostCard/PostCard.js [hunk 6]
```diff
@@ -10,39 +10,43 @@ import {
...
               </h2>
               <DecoLine />
             </DateBox>
-            <StyledParagraph>{moment.day}</StyledParagraph>
-            <h3>{moment.title}</h3>
-            {moment.img !== '' ? 
-            <img src={moment.img} width="150px" alt = "moment"></img> : ''}
+            <StyledParagraph>{post.day}</StyledParagraph>
+            <h3>{post.title}</h3>
+            {post.img !== '' ? <img src={post.img} width="150px" alt="moment"></img> : ''}
```
[the remark this answers is not in the document]
Comment (Sanae-G) [reply client/src/components/PostCard/PostCard.js:43]: Thank you very much. I forgot this approach! :)

File: client/src/components/ScrollUpButton.js (hunk 3 above)
Comment (Sanae-G) [reply client/src/components/ScrollUpButton.js:17]: Yes it does! :D At least on firefox! xD I will deploy my app today! ^^

**Pull request merged**